README.md

```diff
@@ -1,37 +1,12 @@
 # Medicines and Healthcare products Regulatory Agency
 
+## Contents
+
 This is a monorepo containing the following domains:
 
 - [Medicines Information Portal](./medicines)
```

Comment:
This area of the repo is named differently elsewhere, notably in `medicines/README.md`, where it's now called _MHRA Products Portal_. If the official name is _MHRA Products Portal_, can we change this line to reflect that?